state-recovery/besu-plugin/src/main/kotlin/linea/staterecovery/plugin/LineaStateRecoveryPlugin.kt

```diff
@@ -29,7 +30,14 @@ fun <T : BesuService> ServiceManager.getServiceOrThrow(clazz: Class<T>): T {
 
 open class LineaStateRecoveryPlugin : BesuPlugin {
   private val log: Logger = LogManager.getLogger(LineaStateRecoveryPlugin::class.java)
-  private val vertx = Vertx.vertx()
```

Comment:
I missed this piece. Is it impossible to get access to Besu's Vertx from a plugin?

Reply:
Yes it's not possible!  Already reached to besu team they prefer to keep it this way.